local-test-infra/fsdiff-deployment.yaml

```diff
@@ -0,0 +1,62 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: node-master
+  namespace: {{ NAMESPACE_TO_GENERATE }}
```

Comment:
Please add the label "test-network-function.com/generic: fs_diff_master" also to the deployment's metadata so this deployment can be easily filtered out by the TNF.

Reply:
Done